drizzlepac/pars/hap_pars/mvm_parameters/acs_hrc_index.json

```diff
@@ -4,8 +4,6 @@
     },
     "astrodrizzle": {
         "acs_hrc_any_n2": "acs/hrc/acs_hrc_astrodrizzle_any_n2.json",
-        "acs_hrc_any_n4": "acs/hrc/acs_hrc_astrodrizzle_any_n4.json",
-        "acs_hrc_any_n6": "acs/hrc/acs_hrc_astrodrizzle_any_n6.json",
```

Comment:
Line 8 ("n6") should not be deleted.  The file should also not be deleted.

Reply:
For the MVMs, these are never used. See [Line 174](https://github.com/spacetelescope/drizzlepac/blob/8402dd205dbad7af13da15af4e2e5bc266c47dd0/drizzlepac/haputils/config_utils.py#L174) of config_utils.py

Reply:
Sorry. Thought I was in the SVMs.